WordPress/Classes/Utility/BackgroundTasks/WeeklyRoundupBackgroundTask.swift

```diff
@@ -357,6 +357,14 @@ class WeeklyRoundupBackgroundTask: BackgroundTask {
 
     func run(onError: @escaping (Error) -> Void, completion: @escaping (Bool) -> Void) {
 
+        // This will no longer run for WordPress as part of JetPack migration.
+        // This can be removed once JetPack migration is complete.
+        if AppConfiguration.isWordPress {
```

Comment:
Note: It should check `shouldFilterWordPressNotifications` if we don't remove this feature entirely from WordPress

Reply:
I've updated this to use NotificationFilteringService